src/db/migrations/20220811170457_create_refresh_token_table.ts

```diff
@@ -1,17 +1,13 @@
-import { Knex } from "knex";
-
+import { Knex } from 'knex';
 
 export async function up(knex: Knex): Promise<void> {
-    return knex.schema.createTable('refresh_token', (table) => {
-        table.string('refreshToken').notNullable();
-        table.string('id').notNullable();
-        table.string('expiresAt').notNullable();
-      });
+  return knex.schema.createTable('refresh_token', (table) => {
+    table.string('refreshToken').notNullable();
+    table.string('id').notNullable();
```

Comment:
Numerical Id would be better in my opinion. They are lot more fast in terms of query speed compared to string.

Reply:
ok.